tokio-stream/src/wrappers/broadcast.rs

```diff
@@ -0,0 +1,38 @@
+use crate::Stream;
+use async_stream::try_stream;
+use std::pin::Pin;
+use tokio::sync::broadcast::error::RecvError;
+use tokio::sync::broadcast::Receiver;
+
+use std::task::{Context, Poll};
+
+/// A wrapper around [`Receiver`] that implements [`Stream`]. Achieved by using the [`async-stream`] crate.
+///
+/// [`Receiver`]: struct@tokio::sync::broadcast::Receiver
+/// [`Stream`]: trait@crate::Stream
+/// [`async-stream`]: https://docs.rs/async-stream
+#[derive(Debug)]
+pub struct BroadcastStream<T: Clone> {
+    inner: Pin<Box<dyn Stream<Item = Result<T, RecvError>> + Send + Sync >>,
+}
+
+impl<T: Clone + Unpin + 'static + Send + Sync> BroadcastStream<T> {
```

Comment:
We can let the bounds stay for now. If we want to do some of the unsafe I mentioned, it can be done in a follow-up release.